controllers/past-grants/views/grant-results.njk

```diff
@@ -1,40 +1,37 @@
 {% from "components/promo-card/macro.njk" import promoCard %}
 {% from "components/split-nav/macro.njk" import splitNav with context %}
+{% from './helpers.njk' import pgLink, buildLocationList with context %}
 
 {% macro grantResultItem(grant, options) %}
     <div class="u-margin-bottom-s{% if options.wrapperClass %} {{ options.wrapperClass }}{% endif %}" data-score="{{ grant._textScore }}">
         {% set subtitle %}
             £{{ grant.amountAwarded | numberWithCommas}} to
-            <strong>{{ grant.recipientOrganization[0].name }}</strong>
+            <strong>{{ grant.recipientOrganization[0].name | striptags }}</strong>
```

Comment:
Not really sure why this was needed but Vue seemed to be adding `&nbsp;` in place of spaces (I checked, this isn't in the original data) on this page: `/funding/search-past-grants-alpha/grant/0030008252`:
![image](https://user-images.githubusercontent.com/394376/47168038-ce052980-d2f7-11e8-9fcd-743e9c90aee8.png)

This was breaking the grid layout.